Header header
int32 ac_id
float64 crosstrack_error  # distance(position(x,y), expected_position(x,y)) (at time)
float64 altitude_error    # altitude(z) - expected_altitude(z) (at time)
float64 heading_error     # heading(psi) - desired_heading(psi) (at time)
